app/08/assets/js/article/collection/articles.js

```diff
@@ -0,0 +1,18 @@
+var app = app || {};
+
+app.ArticleCollection = Backbone.Collection.extend({
+	model : app.ArticleModel,
+
+	initialize: function() {
+    this.fetchAllArticles()
+  },
+  fetchAllArticles: function () {
```

Comment:
@chiefGui 
https://github.com/BackboneJsBrasil/Backbone4Dummies/commit/a69bca2b322043716fca18bfc5b3dd2f1b46bd52#diff-dd9af97846cc3f450add75fa1adca233R9

It is correct, I don't know why is showing in this way here...